src/ui/public/agg_types/buckets/ip_range.js

```diff
@@ -15,10 +15,9 @@ export default function RangeAggDefinition(Private) {
     createFilter: createFilter,
     getKey: function (bucket, key, agg) {
       if (key) return key;
-
-      const from = _.get(bucket, 'from', '*');
-      const to = _.get(bucket, 'to', '*');
-      return `${from}-${to}`;
+      const from = _.get(bucket, 'from', '-Infinity');
+      const to = _.get(bucket, 'to', 'Infinity');
+      return `${from} to ${to}`;
```

Comment:
There shouldn't be any BWC issue. `getKey()` just needs to return a unique string per bucket